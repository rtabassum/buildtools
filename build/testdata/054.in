# If-else clauses

if True:
  pass
elif False:
  pass

if foo in bar:
 for f in foo:
  # loop
  f()
else if True:
  a = b

else:
  if False:

    #assign
    b=a
  else:

   (a,b)=(b,a)

if foo:
  bar
