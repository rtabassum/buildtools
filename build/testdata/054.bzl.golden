# If-else clauses

if True:
    pass
elif False:
    pass

if foo in bar:
    for f in foo:
        # loop
        f()
elif True:
    a = b

elif False:
    #assign
    b = a
else:
    (a, b) = (b, a)

if foo:
    bar
